recipes/renode-cli/recipe.yaml

```diff
@@ -0,0 +1,86 @@
+context:
+  name: renode-cli
+  version: "1.15.3"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  - url: https://github.com/renode/renode/releases/download/v${{ version }}/renode_${{ version }}_source.tar.xz
+    sha256: 40c50afe8db86a9c63f4a86b93116a6babd43bf241cfaac47b2c0d2cbc2b15a8
+    patches:
+      - patches/update-IronPython-Renode_NET.csproj.patch
```

Comment:
I think it would be better to convert this patch to a simple `sed` replacement since it's a temporary fix for a dependency version that will probably be updated in a later release. We can add a comment above the `sed` statement explaining that it can be removed once there is a new release that fixes the version.

Reply:
Done